.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/c/sdks/auth0-spa-js/71
+    about: Discuss this SDK in the Auth0 Community forums
+  - name: FAQ
+    url: https://github.com/auth0/auth0-spa-js/blob/master/FAQ.md
+    about: Read the FAQ to get answers to common issues
+  - name: Library Documentation
```

Comment:
The library docs have a bit more context around the API and the use cases, and links off to various places (migration guide from Auth0.js, etc), whereas the API docs just describe the API and different properties in a list. I don't think either of them by themselves would help the developer solve specific problems, they are just useful references.

Perhaps we link to both or neither of them to avoid confusion?

Reply:
I was wondering about both, don't want to overload with buttons. However, 7 +/- 2 should be fine.